.circleci/config.yml

```diff
@@ -13,6 +13,12 @@ executors:
     resource_class: medium
     environment:
       NODE_OPTIONS: --max_old_space_size=3072
+  node-linux-medium:
+    machine:
+      image: ubuntu-2204:2024.04.4
```

Comment:
We're in an Ubuntu LTS transition period.  I think we might as well go with the next version, which is
`ubuntu-2404:2024.05.1`